### ISSUE TITLE
Feature Request: URL links in HTML mail notifications

### ISSUE THREAD

**Issue by papimla** [wekan/wekan#3118]
When HTML mails notifications are sent, the URLs to the corresponding card/board are not links, the are plain text. As a result mail clients such as [evolution](https://gitlab.gnome.org/GNOME/evolution), thunderbird or [roundcube](https://roundcube.net/) do not allow the user to click on them which is inconvenient. Other email clients such as Mail on MacOS parse the content of the mail and transform it into a URL.

It would be a nice improvement for the links to the wekan cards to be **&lt;a href="...">...&lt;/a>** instead of plain text when [htmlEnabled](https://github.com/wekan/wekan/blob/master/server/notifications/email.js#L16-L18) is set.

**Comment by papimla** [comment]
Looking [at the code](https://github.com/wekan/wekan/blob/master/server/notifications/email.js#L16-L18) it appears notifications are not formatted as HTML when RICHER_CARD_COMMENT_EDITOR is not set.

**Comment by papimla** [comment]
After activating RICHER_CARD_COMMENT_EDITOR with

    snap set wekan richer-card-comment-editor=true

the mail notification still is plain text.

**Comment by papimla** [comment]
I updated the title of the issue because the feature request actually is to change the links to the wekan cards from being plain text to HTML **&lt;a href="...">...&lt;</a>** when htmlEnabled is set.

**Comment by papimla** [comment]
Users interested in this could +1 to show this is not just me. Or maybe it's just me :-)

**Comment by papimla** [comment]
I'm still hoping that would help someone else.

**Comment by qchuchu** [comment]
Hi ! 

With my friend @triskel78 we are interested with this subject ! We managed to successfully install the project on our laptops, but we were wondering a way to simulate email sending, if you had any clue about this would help us to move further :)

**Comment by xet7** [comment]
@qchuchu @triskel78

Like this? https://github.com/mailslurper/mailslurper

For sending email, Wekan uses https://atmospherejs.com/meteor/email

For related packages, you can search from https://atmospherejs.com

Finding email related code:
```
cd wekan
./find.sh email
```

Related https://github.com/wekan/wekan-snap/issues/78 #2375 

Related https://github.com/wekan/wekan/issues/2414#issuecomment-530119469

**Comment by qchuchu** [comment]
Hey all,

I deep dived into the code to take a look at this new feature improvement.

Actually, changing the card link to an anchor tag (```<a href="${params.url}">${params.url}</a>```) could do the fix for forcing the card link to be a link. 

For the other links, it's mandatory to parse the text with a regex. However, I think that the better solution would be directly to force user to create anchor tags through the richer card comment editor (the insert feature seems to be commented in the code).

I'm going to try to add links in the user input directly, I definitely think it should be better.

**Comment by tromlet** [comment]
Yep. We think this would be a nice change, to have clickable links in WeKan emails.

**Comment by tromlet** [comment]
the issue with that is that our users were running into #2890, so long comments and copying and pasting were interrupting the workflow for a lot of our users. Plus we're nerds, and like Markdown - I suppose we could manually add links in Markdown a la [this](https://static.boredpanda.com/blog/wp-content/uploads/2016/02/big-cute-eyes-cat-black-scottish-fold-gimo-1room1cat-49.webm) if that would make them happy in our emails - but then the generated link to the card itself (in the email) doesn't appear clickable, either. :(
